### PR TITLE
Consistent proposal v3

### PR DESCRIPTION
**Pull request by VelinGeorgiev**

### CODE REVIEW

File: commands.yaml [hunk 1]
```diff
@@ -1,83 +1,70 @@
...
-graph o365group user add:
-graph teams user add:
+flow environment get:
+flow environment list:
+flow export:
+flow get:
+flow list:
+flow run get:
+flow run list:
+o365groups groupsetting add:
```
Comment (VelinGeorgiev) [inline commands.yaml:18]: @waldekmastykarz  how about the o365groups namespace, is it a good fit? 

`o365groups groupsetting add:` or `o365groups group setting add:` or `o365groups setting add:`?

File: commands.yaml [hunk 2]
```diff
@@ -1,83 +1,70 @@
...
+o365groups groupsetting get: 
+o365groups groupsetting list:
+o365groups groupsetting remove:
+o365groups groupsetting set:
+o365groups groupsettingtemplate get:
+o365groups groupsettingtemplate list:
+o365groups add:
```
Comment (VelinGeorgiev) [inline commands.yaml:25]: `o365groups add:` or `o365groups group add:`? @waldekmastykarz

File: commands.yaml [hunk 3]
```diff
@@ -1,83 +1,70 @@
...
-graph teams user remove:
-graph teams user set:
-graph user get:
-graph user list:
-graph user sendmail:
+aad siteclassification disable:
+aad siteclassification enable:
+aad siteclassification get:
+aad siteclassification set:
+teams team add:
```
Comment (VelinGeorgiev) [inline commands.yaml:38]: I've added `teams team add:` instead of `teams add:`? Should we do that? @waldekmastykarz

File: commands.yaml [hunk 4]
```diff
@@ -1,83 +1,70 @@
...
+teams channel message get:
+teams message list:
+teams messagingsettings list:
+teams messagingsettings set:
+teams tab list:
+teams unarchive:
+teams user list:
+teams user remove:
+teams user set:
+aad user get:
```
Comment (VelinGeorgiev) [inline commands.yaml:65]: Added `aad` as per proposal v2

File: commands.yaml [hunk 5]
```diff
@@ -1,83 +1,70 @@
...
+aad user list:
+user sendmail:
```
Comment (VelinGeorgiev) [inline commands.yaml:67]: @waldekmastykarz how about that `user sendmail:` should it be `exchange user sendmail:` or ` outlook user sendmail:` or `email send`?

File: commands.yaml [hunk 6]
```diff
@@ -1,83 +1,70 @@
...
+o365groups groupsettingtemplate list:
+o365groups add:
+o365groups get:
+o365groups list:
+o365groups remove:
+o365groups restore:
+o365groups set:
+o365groups user add:
+teams user add:
 graph schemaextension add:
```
Comment (VelinGeorgiev) [inline commands.yaml:33]: @waldekmastykarz, that should remain `graph` and will be very alone in its own folder for now :)

**Review by VelinGeorgiev — COMMENTED** [review]
@waldekmastykarz  here is proposal v3 with some additional comments, please let me know what do you think. Thanks!

File: commands.yaml (hunk 1 above)
Comment (waldekmastykarz) [reply commands.yaml:18]: o365groups is not a service, so perhaps something that we should put under aad

File: commands.yaml (hunk 2 above)
Comment (waldekmastykarz) [reply commands.yaml:25]: If we put Office 365 groups under add then we could do `aad o365group add`

File: commands.yaml (hunk 3 above)
Comment (waldekmastykarz) [reply commands.yaml:38]: I think it make sense to clarify what the command does and it's also consistent with other Teams commands like `team channel add`, `teams tab add`, etc.

File: commands.yaml (hunk 4 above)
Comment (waldekmastykarz) [reply commands.yaml:65]: Are we getting users directly from AAD? What current command is it mapping to?

File: commands.yaml (hunk 5 above)
Comment (waldekmastykarz) [reply commands.yaml:67]: ...or `mail send`, mail being the service with which you interact

File: commands.yaml (hunk 3 above)
Comment (waldekmastykarz) [reply commands.yaml:38]: Thinking of it, in Flow we have `flow list` and not `flow flow list` so why should Teams be different? 🤔

Comment (VelinGeorgiev) [reply commands.yaml:38]: I agree with the short version as currently is.

File: commands.yaml (hunk 4 above)
Comment (VelinGeorgiev) [reply commands.yaml:65]: It is currently `graph` so maybe the underlying service is aad

File: commands.yaml (hunk 5 above)
Comment (VelinGeorgiev) [reply commands.yaml:67]: I agree with `mail send` fits just perfect

Comment (waldekmastykarz) [reply commands.yaml:67]: lol, see my other comment 🤦‍♂

**Pull request closed without merging**